### PR TITLE
Semantic release

### PR DESCRIPTION
**Pull request by nickbalestra**
📦  💥  semantic-release ftw

ISSUES CLOSED: #32

### CODE REVIEW

**Comment by nickbalestra** [comment]
Blocked by https://github.com/conventional-changelog/validate-commit-msg/issues/94

File: package.json [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
     "postinstall": "lerna bootstrap",
     "clean": "lerna clean",
     "coverage": "codecov",
     "test": "jest",
-    "publish": "lerna publish --exact",
-    "precommit": "lint-staged"
+    "commitmsg": "validate-commit-msg",
+    "commit": "git-cz",
+    "precommit": "lint-staged",
+    "semantic-release": "lerna-semantic-release pre && lerna-semantic-release post && lerna-semantic-release perform"
```
Comment (nickbalestra) [inline package.json]: order is wrong: pre -> peform -> post

**Comment by nickbalestra** [comment]
I've unblocked this by releasing v2.12.3 of `conventional-changelog/validate-commit-msg`.
We'll just have to merge this, and watch if it actually works...

gif

**Comment by matteofigus** [comment]
Very well! 🤣 go for it

**Pull request merged**